src/pyhf/infer/calculators.py

```diff
@@ -192,10 +194,11 @@ def __init__(
         Args:
             data (:obj:`tensor`): The observed data.
             pdf (~pyhf.pdf.Model): The statistical model adhering to the schema ``model.json``.
-            init_pars (:obj:`tensor`): The initial parameter values to be used for fitting.
-            par_bounds (:obj:`tensor`): The parameter value bounds to be used for fitting.
-            fixed_params (:obj:`tensor`): Whether to fix the parameter to the init_pars value
-              during minimization.
+            init_pars (:obj:`tensor` of :obj:`float`): The starting values of the model parameters for minimization.
+            par_bounds (:obj:`tensor` of shape Nx2): The extrema of values the model parameters
+                are allowed to reach in the fit.
```

Comment:
```suggestion
        par_bounds (:obj:`tensor`): The extrema of values the model parameters
         are allowed to reach in the fit.
         The shape should be ``(n, 2)`` for ``n`` model parameters.
```

Reply:
can't add this, the spacing/indenting is wrong.